src/main/scala/org/virtuslab/yaml/yaml.scala

```diff
@@ -7,8 +7,8 @@ import org.virtuslab.internal.YamlError
 
 import scala.deriving._
 
-def deriveEncoder[T](using m: Mirror.Of[T]) = YamlEncoder.derived
-def deriveDecoder[T](using m: Mirror.Of[T]) = YamlDecoder.derived
+// def deriveEncoder[T](using m: Mirror.Of[T]) = YamlEncoder.derived
+// def deriveDecoder[T](using m: Mirror.Of[T]) = YamlDecoder.derived
```

Comment:
If you commented any code,  please write a short explanatory comment or just remove code.